pkg/kapp/resourcesmisc/custom_waiting_resource.go

```diff
@@ -131,6 +131,7 @@ func (s CustomWaitingResource) IsDoneApplying() DoneApplyState {
 	}
 
 	unblockChangeMsg := ""
+	actualMsg := "No failing or successful conditions found"
```

Comment:
Maybe just `message` instead of `actualMsg`? 
We are assigning the variable with a custom message which indicated that it is just not "actual" 🤔